tardis/io/parsers/__init__.py

```diff
@@ -1,3 +1,7 @@
+"""
+A collection of parsers to read data of different formats. 
```

Comment:
As a second part of the docsting, maybe write what you mean as different formats.

Reply:
Thanks for the quick review. Extended summary added. Please review again :)